cloud_formation/configs/api.py

```diff
@@ -165,21 +182,14 @@ def create_config(bosslet_config, db_config={}):
                                user_data=parsed_user_data,
                                min=const.ENDPOINT_CLUSTER_MIN,
                                max=const.ENDPOINT_CLUSTER_MAX,
-                               elb=Ref("EndpointLoadBalancer"),
+                               #elb=Ref("EndpointLoadBalancer"),
```

Comment:
Why keep the old values still commented out vs just deleting them?

Reply:
Was careless and didn't catch these during cleanup.